ai_services/anomaly_detection/data_preprocessing_examples/oci_data_flow_based_examples/example_code/driver-demoable.json

```diff
@@ -130,11 +122,25 @@
             }
         }
     ],
-    "outputDestination": {
+    "stagingDestination": {
         "combinedResult":"M1",
         "type": "object-storage",
         "namespace":"<namespace>",
         "bucket": "<bucket>",
         "folder": "<path_to_file>"
+    }, 
+    "outputDestination": {
+        "type": "object-storage",
+        "namespace":"<namespace>",
+        "bucket": "<bucket>",
+        "folder": "<path_to_file>"
+    },
+    "serviceApiConfiguration": {
```

Comment:
[Minor] This will help with the generalization in future given that the difference in calling different AI services is the input and the client being used.
```suggestion
    "serviceApiConfiguration": {
        "clientType": "AnomalyDetectionClient",
        "serviceEndpoint": null,
        "profileName": "DEFAULT",
        "projectId": "<project_id>",
        "compartmentId": "<compartment_id>"
    }
```

Reply:
I don't think the new way is better - what if the customer wants to have multiple post processing steps? I believe your original expressions is better, like
```
"serviceApiConfiguration": {
       "anomalyDetection": {...},
       "focastingService":{...}
    }
```

Reply:
That's a valid concern. Just wondering if same (training/inference) data can be used for multiple AI-Services at the same time. I think that it is good use-case but cannot be a real-world scenario and so it might not be good to consider.

Reply:
Users may not only want to do training/inferencing, but they can do whatever they want. For example, they might want to send a message to OSS with an OSS client, or they might want to piping the processed data into some local database, so they need configuration for that database too. We definitely shouldn't remove the freedom from user.